src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/HdcLicence.kt

```diff
@@ -168,11 +172,18 @@ class HdcLicence(
   standardConditions = standardConditions.toMutableList(),
   additionalConditions = additionalConditions.toMutableList(),
   bespokeConditions = bespokeConditions.toMutableList(),
-  responsibleCom = responsibleCom,
   updatedBy = updatedBy,
 ),
   HasElectronicMonitoringResponseProvider,
-  HdcCase {
+  HdcCase,
+  HasCom {
+
+  @RequiresCom("Does a Hdc licence always need a COM set?")
```

Comment:
yes it does, you can't start to create a licence without one - it's the COM who creates it